entity-framework/ef6/fundamentals/databinding/wpf.md

```diff
@@ -247,11 +247,11 @@ Add the classes that are defined in the model as data sources for this WPF appli
 
     ![Data Sources](~/ef6/media/datasources.png)
 
--   Select the **Category **data source and drag it on the form.
+-   Select the **Category **data source and drag it on the form.<!-- extra asterisks -->
```

Comment:
I don't think the asterisks are extra, they are just "misaligned". They should be surrounding the word Category. as in `**Category**`.